tools/gdb/pytorch-gdb.py

```diff
@@ -18,7 +18,12 @@ def __enter__(self) -> None:
                 b.enabled = False
                 self.disabled_breakpoints.append(b)
 
-    def __exit__(self, etype: Any, evalue: Any, tb: Any) -> None:
+    def __exit__(
+        self,
+        etype: type[BaseException] | None,
```

Comment:
We don't support this so you need to use the deprecated alias:
```suggestion
        etype: Type[BaseException] | None,
```
`from typing import Type`